tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -8617,6 +8617,10 @@ ExprResult HLSLExternalSource::LookupVectorMemberExprForHLSL(
     llvm_unreachable("Unknown VectorMemberAccessError value");
   }
 
+  if (colCount > 4) {
+    msg = diag::err_hlsl_vector_member_on_long_vector;
+  }
```

Comment:
nit: see: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements
```suggestion
  if (colCount > 4)
    msg = diag::err_hlsl_vector_member_on_long_vector;
```